pkg/kube/translation/apisix_route.go

```diff
@@ -530,17 +530,8 @@ func (t *translator) translateHTTPRouteV2beta3(ctx *TranslateContext, ar *config
 		route.EnableWebsocket = part.Websocket
 		route.Plugins = pluginMap
 		route.Timeout = timeout
-		pluginConfig := apisixv1.NewDefaultPluginConfig()
-		if len(pluginMap) > 0 {
-			pluginConfigName := apisixv1.ComposePluginConfigName(ar.Namespace, backend.ServiceName)
-			route.PluginConfigId = id.GenID(pluginConfigName)
-			pluginConfig.ID = route.PluginConfigId
-			pluginConfig.Name = pluginConfigName
-			pluginConfig.Plugins = pluginMap
-		} else {
-			if part.PluginConfigName != "" {
-				route.PluginConfigId = part.PluginConfigName
-			}
+		if part.PluginConfigName != "" {
+			route.PluginConfigId = part.PluginConfigName
```

Comment:
We don't use the plugin config name as the id, see https://github.com/apache/apisix-ingress-controller/blob/master/pkg/kube/translation/apisix_pluginconfig.go#L51 for details.

Reply:
ok, I will refactor it right now